src/mapping.ts

```diff
@@ -1,3 +1,81 @@
+import { Address, BigInt, log } from "@graphprotocol/graph-ts";
 import { Transfer } from "../generated/CowProtocolToken/CowProtocolToken";
+import { Holder, Supply } from "../generated/schema";
 
-export function handleTransfer(event: Transfer): void {}
+export const COW_TOKEN = "0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB";
+export const NON_CIRCULATING = [
+  // CoW DAO Safe
+  Address.fromString("0xcA771eda0c70aA7d053aB1B25004559B918FE662"),
+  // Vested tokens (vCOW token address)
+  Address.fromString("0xD057B63f5E69CF1B929b356b579Cba08D7688048"),
+  // TODO - add GnosisDAO Vesting contract TBD
+];
```

Comment:
Maybe it's always a good idea to add address(0x) to the non_cirulating supply. This would allow us to burn token later with ease.

Reply:
This would clean up the logic below involving minting and burning... Although I would have to carefully think about how it affects the supply changing logic...

Reply:
The zero address is a special case that should not be part of this list. In particular it different in two ways:

1. The zero address balance is always negative.
2. The zero address is excluded from the holders. 

We treat mint and burn events as those coming to/from the zero address which is a different scenario/special case of total and circulating supply changes.

So we won't be including this address in the non-circulating list.